### PR TITLE
[docs] Added info about StorageClass at specs

### PR DESCRIPTION
**Pull request by mortelumina98**
## Description
Added info about StorageClass at specs.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: docs
type: chore
summary: Added info about StorageClass at specs.
impact_level:  low
```

### CODE REVIEW

File: modules/030-cloud-provider-yandex/openapi/config-values.yaml [hunk 1]
```diff
@@ -20,11 +20,15 @@ properties:
       default:
         type: string
         description: |
```
Comment (mortelumina98) [inline modules/030-cloud-provider-yandex/openapi/config-values.yaml:22]:
```suggestion
        description: |-
```

File: modules/030-cloud-provider-yandex/openapi/doc-ru-config-values.yaml [hunk 2]
```diff
@@ -8,8 +8,13 @@ properties:
           Полные имена (или regex-выражения имен) StorageClass, которые не будут созданы в кластере.
       default:
         description: |
```
Comment (mortelumina98) [inline modules/030-cloud-provider-yandex/openapi/doc-ru-config-values.yaml:10]:
```suggestion
        description: |-
```

File: modules/030-cloud-provider-azure/openapi/config-values.yaml [hunk 3]
```diff
@@ -14,11 +14,16 @@ properties:
       default:
         type: string
         description: |
```
Comment (mortelumina98) [inline modules/030-cloud-provider-azure/openapi/config-values.yaml:16]:
```suggestion
        description: |-
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: modules/030-cloud-provider-aws/openapi/doc-ru-config-values.yaml [hunk 4]
```diff
@@ -34,8 +34,14 @@ properties:
...
       default:
-        description: |
-          Имя StorageClass, который будет использоваться в кластере по умолчанию:
-            * Если не задан, фактическим StorageClass по умолчанию будет:
-              * присутствующий в кластере произвольный StorageClass с default-аннотацией;
-              * лексикографически первый StorageClass из создаваемых модулем.
+        description: |-
+          Имя StorageClass, который будет использоваться.
+          
+          Если параметр не задан, фактическим StorageClass по умолчанию будет:
```
Comment (z9r5) [inline modules/030-cloud-provider-aws/openapi/doc-ru-config-values.yaml:40]:
```suggestion
          Если параметр не задан, фактический StorageClass по умолчанию будет:
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/stage'...

**Pull request merged**